docs/addons/istio-base.md

```diff
@@ -0,0 +1,64 @@
+# Istio Base Add-on
+
+The Base add-on adds support for Istio base chart which contains cluster-wide resources and CRDs used by the Istio control plane to an EKS cluster.
+
+## Usage
+
+Add the following as an add-on to your main.ts file to add Istio Base to your cluster
+
+```typescript
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.IstioBaseAddOn();
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+To validate that installation is successful run the following command:
+
+```bash
+$ kubectl get crd -n istio-system
+NAME                                         CREATED AT
+authorizationpolicies.security.istio.io      2022-05-05T02:16:23Z
+destinationrules.networking.istio.io         2022-05-05T02:16:23Z
+eniconfigs.crd.k8s.amazonaws.com             2022-05-05T02:04:10Z
+envoyfilters.networking.istio.io             2022-05-05T02:16:23Z
+gateways.networking.istio.io                 2022-05-05T02:16:23Z
+istiooperators.install.istio.io              2022-05-05T02:16:23Z
+peerauthentications.security.istio.io        2022-05-05T02:16:23Z
+proxyconfigs.networking.istio.io             2022-05-05T02:16:23Z
+requestauthentications.security.istio.io     2022-05-05T02:16:23Z
+securitygrouppolicies.vpcresources.k8s.aws   2022-05-05T02:04:12Z
+serviceentries.networking.istio.io           2022-05-05T02:16:23Z
+sidecars.networking.istio.io                 2022-05-05T02:16:23Z
+telemetries.telemetry.istio.io               2022-05-05T02:16:23Z
+virtualservices.networking.istio.io          2022-05-05T02:16:23Z
+wasmplugins.extensions.istio.io              2022-05-05T02:16:23Z
+workloadentries.networking.istio.io          2022-05-05T02:16:23Z
+workloadgroups.networking.istio.io           2022-05-05T02:16:23Z
+```
+
+Note that the all the CRDs are created in the `istio-system` namespace.
```

Comment:
this is correct with respect to the default add-on configuration. However, unlikes the istio-control-plane add-on, namespace for this add-on can also be passed in by the user. If that is allowed, then this statement should be changed to the "provided" namespace (istio-system by default). 
If it is not allowed, let's discuss.

Reply:
Fixed